WordPress/WordPressTest/CoreDataMigrationTests.m

```diff
@@ -602,4 +751,20 @@ - (NSManagedObject *)insertDummyPostInContext:(NSManagedObjectContext *)context
     return post;
 }
 
+- (void)cleanModelObjectClassnames:(NSManagedObjectModel *)model
+{
+    // NOTE: Suppose the following scenario...
+    //  -   `Model N`'s Entity X contains `Attribute A`
+    //  -   `Model N+1`'s Entity X *class* implementation differs from the one bundled in N.
+    //      Added / Removed Attributes. It could differ vastly!
+    //
+    // Problem is... whenever we test `Model N` or `Model N+1`, Core Data will always load the latest
+    // NSManagedObject subclass. This could prove troublesome in a variety of scenarios.
+    // For that reason, we're implementing this helper, which will nuke NSMO's classnames.
+    //
```

Comment:
The reason for doing this, precisely, is: `Blog` used to have attributes, suck as `privacy`, right there.
In this PR, all of those got moved to the `BlogSettings` relationship.

Problem: If you load a previous Data Model... `Blog` lacks this `Settings` property, still, the `Blog` instances are mapped to the latest `Blog`'s NSManagedObject subclass, and everything fails miserably.

I believe this is the reason why Core Data's heavyweight migrations only instantiate `NSManagedObject` instances (it never instantiates the actual subclass).

I'll update the comment with extra details, thank you!